### PR TITLE
[update] renew page: move field order and autofocus

### PR DESCRIPTION
**Pull request by blckbx**
This PR moves field order on renew page from 
- TS server
- wg pubkey input
to
- wg pubkey input
- TS server

and sets autofocus for field "wg pubkey input" on page switch.
Additionally this PR includes some code formatting improvements using `// prettier-ignore` for telegram messages and replaces "Oregon" with "OR" on tooltip of US West VPN for better readability.

### CODE REVIEW

**Review by TrezorHannes — COMMENTED** [review]
Probably good to add the review from @ziggie1984

File: frontend/client/src/App.js [hunk 1]
```diff
@@ -720,7 +721,7 @@ function App() {
                     pointerEvents={"none"}
                     Cursor={"not-allowed"}
                   />
-
+                  <p className="price">{server}</p>
```
Comment (ziggie1984) [inline frontend/client/src/App.js:724]: Is the className price right here or does it even matter ?

File: frontend/client/src/App.js [hunk 2]
```diff
@@ -749,16 +750,17 @@ function App() {
                           />
                         </Button>
                       </InputGroup>
+                      {/*
```
Comment (ziggie1984) [inline frontend/client/src/App.js:753]: Maybe add a more detailed comment why it was removed or otherwise remove it completely ? (for our future selves ?

**Review by ziggie1984 — APPROVED** [review]
Thank you for improving the design, LGTM.

File: frontend/client/src/App.js (hunk 2 above)
Comment (blckbx) [reply frontend/client/src/App.js:753]: will remove 👍

File: frontend/client/src/App.js (hunk 1 above)
Comment (blckbx) [reply frontend/client/src/App.js:724]: class could be renamed, used for consistency of color and font style

**Review by ziggie1984 — APPROVED** [review]

**Pull request merged**